CONTRIBUTING.md

```diff
@@ -0,0 +1,139 @@
+# Guidance on how to contribute
+
+> All contributions to this project will be released under the Apache License, Version 2.0.
+> By submitting a pull request or filing a bug, issue, or
+> feature request, you are agreeing to comply with this waiver of copyright interest.
+> Details can be found in our [LICENSE](LICENSE).
+
+Hey there! We’re really happy you’ve found your way here. Dataverse&reg; is a community project that is developed by people from all over the world. We appreciate any help.
+
+Before you start working on something we would suggest talking to us in the [TODO: discussion forum](#) or asking a question on [TODO: GoogleGroups/Zulip](#). You can also contact us via the chat widget on the [TODO: main Dataverse website](#).
```

Comment:
Just pointing out that there are a number of TODO's in the doc, including this one.

Reply:
I addressed all the TODOs in the CONTRIBUTING.md file. Well, I mostly rewrote it, taking the CONTRIBUTING.md file from the main Dataverse repository as a reference. I also added the DEVELOPER_GUIDE.md file for more information about code contributions